apps/mobile/src/components/Feed/Posts/PostListCaption.tsx

```diff
@@ -0,0 +1,28 @@
+import { Text, View } from 'react-native';
+import { graphql, useFragment } from 'react-relay';
+
+import { PostListCaptionFragment$key } from '~/generated/PostListCaptionFragment.graphql';
+
+type Props = {
+  feedPostRef: PostListCaptionFragment$key;
+};
+
+export function PostListCaption({ feedPostRef }: Props) {
+  const feedPost = useFragment(
+    graphql`
+      fragment PostListCaptionFragment on Post {
+        __typename
+        caption
+      }
+    `,
+    feedPostRef
+  );
+
+  const { caption } = feedPost;
+
+  return (
+    <View className="px-4 pb-4">
```

Comment:
I think this needs a text color for dark mode

![Screenshot 2023-07-28 at 15 26 10](https://github.com/gallery-so/gallery/assets/80802871/f3122a8e-6127-4704-8010-54fb719e4698)

Reply:
Missed that! Updated it